docs/content/time_dependent.md

```diff
@@ -43,3 +45,39 @@ print(out_state)
 ```
 
 Note that Qadence makes no assumption on units. The unit of passed duration value $\tau$ must be aligned with the units of other parameters in the time-dependent generator so that the integral of generator $\overset{\tau}{\underset{0}{\int}}\mathcal{\hat{H}}(t){\rm d}t$ is dimensionless.
+
+# Noisy time-dependent Hamiltonian evolution
+
+To perform noisy time-dependent Hamiltonian evolution, one needs to pass a list of noise operators to the `noise_operators` argument in `HamEvo`. They correspond to the jump operators used within the time-dependent Schrodinger equation solver method `SolverType.DP5_ME`:
+
+```python exec="on" source="material-block" session="getting_started"
+from qadence import X, Y, HamEvo, TimeParameter, FeatureParameter, run
+from pyqtorch.utils import SolverType
+import torch
+
+# Simulation parameters
+ode_solver = SolverType.DP5_ME  # time-dependent Schrodinger equation solver method
+n_steps_hevo = 500  # integration time steps used by solver
+
+# Define block parameters
+t = TimeParameter("t")
+omega_param = FeatureParameter("omega")
+
+# Arbitrarily compose a time-dependent generator
+td_generator = omega_param * (t * X(0) + t**2 * Y(1))
+
+# Create parameterized HamEvo block
+noise_operators = [X(0) + Y(0)]
+hamevo = HamEvo(td_generator, t, noise_operators = noise_operators)
+
+values = {"omega": torch.tensor(10.0), "duration": torch.tensor(1.0)}
+
+config = {"ode_solver": ode_solver, "n_steps_hevo": n_steps_hevo}
+
+out_state = run(hamevo, values = values, configuration = config)
+
+print(out_state)
+```
+
+!!! warning "Noise operators definition"
+    Note it is not possible to define `noise_operators` with parametric operators. If you want to do so, we recommend obtaining the tensors via run and set `noise_operators` using `MatrixBlock`. Also, `noise_operators` should have the same or a subset of the qubit support of the `HamEvo` instance.
```

Comment:
Two questions:
- I don't understand the suggestion to "obtain the tensors via run and set `noise_operators` using `MatrixBlock`" -- why is that similar to using parametric operators?
- Regarding the qubit support, what is the current behaviour regarding e.g. passing `X(0)` as a jump operator? Will it only affect qubit 0, or all qubits? 

In Pulser, only single-qubit jump operators are possible, and currently the behaviour is set to applying that same operator to all the qubits. To get that in Qadence, would one need to pass `[X(i) for i in range(n_qubits)]` ?

Reply:
@jpmoutinho 

- My bad, I meant using `block_to_tensor` with the parameter values on the operator to convert it to a matrix that should be passed to a MatrixBlock.
- Similarly to applying an X gate to qubit 0, which will be converted to a matrix with filled identity to be applied with mesolve.

Reply:
For Pulser, I guess passing `[X(i) for i in range(n_qubits)]` would be the same, but seems restrictive no?

Reply:
What I mean is that in Pulser only single-qubit jump operators are possible AND they automatically apply to all qubits. So if you pass X, you are essentially passing X(0), X(1), X(2)...

I just wanted to clarify if these restrictions are NOT in place in Qadence / PyQ currently.
- So if you pass X(0), it literally means that only qubit 0 will have a noise effect. 
- If someone wants to run on PyQ and produce the same effect that they would get in Pulser, they do have to write each individual jump operator affecting each individual qubit, e.g. `[X(i) for i in range(n_qubits)]`.

Correct?

Reply:
I think that is correct, they need to specify `[X(i) for i in range(n_qubits)]`.

Reply:
Can you change the example so it shows the syntax for applying the same operator to each qubit instead? (with an explicit comment that you do need to specify each operator for each qubit).